Tests/SessionManagerTests.swift

```diff
@@ -195,7 +193,7 @@ class SessionManagerTestCase: BaseTestCase {
 
     func testDefaultUserAgentHeader() {
         // Given, When
-        let userAgent = HTTPHeaders.defaultHTTPHeaders["User-Agent"]
+        let userAgent = HTTPHeaders.default["User-Agent"]
```

Comment:
As a separate effort, it might be worth trying to pull the header fields into a strongly typed object instead of using magic strings. Something like a `HTTPHeader.Field` type.

```swift
struct HTTPHeader {
    struct Field {
        static let userAgent = "User-Agent"
    }
}
```

Etc. etc... This is something we did in our internal networking library and it's been helpful.